requirements.txt

```
@@ -1,6 +1,5 @@
-beautifulsoup4>=4.9.1
 Click>=7.0
 colorama>=0.4.3
-lxml>=4.5.2
+gtrending>=0.3.0
```

```suggestion
gtrending>=0.3.0,<1.0.0
```
This will avoid new users installing 1.0.0 when it is released which by semantic versioning it is backwards-incompatible